node/service/src/service/mod.rs

```diff
@@ -320,18 +320,14 @@ where
 			create_inherent_data_providers: move |parent, ()| {
 				let client_clone = client_clone.clone();
 				async move {
-					let uncles = sc_consensus_uncles::create_uncles_inherent_data_provider(
-						&*client_clone,
-						parent,
-					)?;
 					let timestamp = sp_timestamp::InherentDataProvider::from_system_time();
 					let slot =
 						sp_consensus_babe::inherents::InherentDataProvider::from_timestamp_and_duration(
 							*timestamp,
 							slot_duration,
 						);
 
-					Ok((timestamp, slot, uncles))
```

Comment:
Are we ok to remove uncles here? And why/

Request some rationale here.

Reply:
This only works on author block, like `timestamp::set`.
We can remove it here safely.